housing/housing.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+from lxml import html
+import requests
+import time
+
+
+HEADERS = {'User-Agent':
+           'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:38.0) Gecko/20100101 Firefox/38.0'}
+
+
+def scrap_housing():
+    place = raw_input('Enter city: ').lower()
+    area = raw_input('Enter area (For multiple area enter ',' seprated values): ').lower()
+    AREAS = area.split(',')
+    for area in AREAS:
+        page = requests.get(
+            'https://housing.com/in/rent/%s/%s'% (place,area),
+            headers=HEADERS)
+
+        tree = html.fromstring(page.text)
+
+        product_url_list = tree.xpath('//div[@class="heading-content information"]/a[@href]/@href')
+        print product_url_list
```

Comment:
if printing for logging purposes then mention that in a comment above this line as a reminder to remove later